### PR TITLE
Implement missing admin method create_client_authz_scope_based_permission() and create_client_authz_policy()

### PR DESCRIPTION
**Pull request by cainotis**
This finishes #383 providing a solution to #370 and allows to create any type of policy

### CODE REVIEW

**Comment by cainotis** [comment]
I believe that something went wrong with the test for Python 3.10 20.0, it didn't pass in the test_token() because the expire time differed by 1. I can't re-run the test without making another commit so I will wait for the reviewers before doing anything unnecessary.
Here is the difference appointed by the test:
  {'refresh_expires_in': 1799} != {'refresh_expires_in': 1800}
  {'expires_in': 299} != {'expires_in': 300}

File: src/keycloak/urls_patterns.py [hunk 1]
```diff
@@ -121,6 +121,9 @@
 URL_ADMIN_CLIENT_AUTHZ_RESOURCE_BASED_PERMISSION = (
     URL_ADMIN_CLIENT_AUTHZ + "/permission/resource?max=-1"
 )
+URL_ADMIN_CLIENT_AUTHZ_SCOPE_BASED_PERMISSION = (
+    URL_ADMIN_CLIENT + "/authz/resource-server/permission/scope?max=-1"
```
Comment (ryshoooo) [inline src/keycloak/urls_patterns.py]: Can we utilize `URL_ADMIN_CLIENT_AUTHZ` instead of `URL_ADMIN_CLIENT`?

Comment (cainotis) [reply src/keycloak/urls_patterns.py]: Yes, I updated the file to use it

**Review by ryshoooo — APPROVED** [review]
Thanks a lot for your contribution :) LGTM!

**Pull request merged**